backend/src/main/java/com/mapbefine/mapbefine/location/domain/Coordinate.java

```diff
@@ -23,17 +22,19 @@ public class Coordinate {
     private static final double LONGITUDE_LOWER_BOUND = 124;
     private static final double LONGITUDE_UPPER_BOUND = 132;
 
-    @Column(columnDefinition = "Decimal(18,15)")
-    private double latitude;
+    /*
+     * 4326은 데이터베이스에서 사용하는 여러 SRID 값 중, 일반적인 GPS기반의 위/경도 좌표를 저장할 때 쓰이는 값입니다.
+     * */
+    private static final GeometryFactory geometryFactory = new GeometryFactory(new PrecisionModel(), 4326);
 
-    @Column(columnDefinition = "Decimal(18,15)")
-    private double longitude;
+    @Column(columnDefinition = "geometry SRID 4326", nullable = false)
+    private Point coordinate;
 
     private Coordinate(double latitude, double longitude) {
-        this.latitude = latitude;
-        this.longitude = longitude;
+        this.coordinate = geometryFactory.createPoint(new org.locationtech.jts.geom.Coordinate(longitude, latitude));
     }
 
+
```

Comment:
```suggestion
    private Coordinate(final Point point) {
        this.coordinate = point;
    }

    public static Coordinate of(double latitude, double longitude) {
        validateRange(latitude, longitude);
        final Point point = geometryFactory.createPoint(new org.locationtech.jts.geom.Coordinate(longitude, latitude));
        
        return new Coordinate(point);
    }
```

이렇게 해서 생성 로직을 정적 팩터리 메서드에서 확인할 수 있는 것은 어떨까요?